www/src/data/sidebars/doc-links.yaml

```diff
@@ -1,128 +1,221 @@
-- title: Quick Start
+- title: Quick start
   items:
-    - title: Getting Started
+    - title: Quick start
       link: /docs/
-    - title: Starters
-      link: /docs/gatsby-starters/
+    - title: Overview of the Docs*
+      link: /docs/overview/
+    - title: Main Tutorial (step-by-step)
+      link: /tutorial/
+- title: Deploying & Hosting
+  items:
+    - title: Preparing site for deployment*
+      link: /docs/preparing-for-deployment/
     - title: Deploying
       link: /docs/deploy-gatsby/
-    - title: Community
-      link: /docs/community/
-    - title: Awesome Gatsby
-      link: /docs/awesome-gatsby/
-- title: Core Concepts
-  items:
-    - title: Building with Components
-      link: /docs/building-with-components/
-    - title: Lifecycle APIs
-      link: /docs/gatsby-lifecycle-apis/
-    - title: Plugins
-      link: /docs/plugins/
-    - title: PRPL Pattern
-      link: /docs/prpl-pattern/
-    - title: Querying Data with GraphQL
-      link: /docs/querying-with-graphql/
+    - title: Path Prefix
+      link: /docs/path-prefix/
+    - title: How Gatsby Works with GitHub pages
+      link: /docs/how-gatsby-works-with-github-pages/
 - title: Guides
   items:
-    - title: Adding a 404 Page
-      link: /docs/add-404-page/
-    - title: Adding Images, Fonts, and Files
-      link: /docs/adding-images-fonts-files/
-    - title: Adding Images to a WordPress Site
-      link: /docs/image-tutorial/
-    - title: Babel.js
-      link: /docs/babel/
-    - title: Browser Support
-      link: /docs/browser-support/
-    - title: Building Apps with Gatsby
+    - title: Environment*
+      link: /docs/environment/
+      items:
+      - title: Basic hardware and software requirements*
+        link: /docs/basic-hardware-software-requirements/
+      - title: Browser Support
+        link: /docs/browser-support/
+      - title: Gatsby on Macs*
+        link: /docs/gatsby-on-macs/
+      - title: Gatsby on Windows
+        link: /docs/gatsby-on-windows/
+    - title: Customization*
+      link: /docs/customization/
+      items:
+        - title: Babel.js
+          link: /docs/babel/
+        - title: Custom webpack config
+          link: /docs/add-custom-webpack-config/
+        - title: Customizing html.js
+          link: /docs/custom-html/
+        - title: Environment variables
+          link: /docs/environment-variables/
+        - title: Structure of Gatsby default starter*
```

Comment:
This is what Amberley wrote--a tour of the default starter that explains how it works, why it comes with what it does, etc. Kyle and I discussed putting her work into the default starter READme and then also somewhere as a doc. Any ideas where it would belong and/or how to title it more clearly?